package.json

```diff
@@ -2,21 +2,28 @@
   "private": true,
   "workspaces": {
     "nohoist": [
-      "**/expo/**",
-      "**/react-native-paper/**",
-      "**/core/**",
+      "**/kbcore/**",
```

Comment:
You're using a non updated version of feature/mobile_layout, could you pull the newest changes and change the PR to that branch?